karabo_bridge/simulation.py

```diff
@@ -44,21 +44,37 @@
 # _SHAPE = (_PULSES, _MODULES, _MOD_X, _MOD_Y)
 
 
-def gen_combined_detector_data(source, tid_counter):
+def gen_combined_detector_data(source, tid_counter, corrected=False):
     gen = {source: {}}
 
     # metadata
     ts = time()
     sec, frac = str(ts).split('.')
-    gen[source]['metadata'] = {
-        'source': source,
-        'timestamp': ts,
-        'timestamp.tid': tid_counter,
-        'timestamp.sec': sec,
-        'timestamp.frac': frac.ljust(18, '0')  # attosecond resolution
+    gen[source] = {
+        'metadata.source': source,
```

Comment:
I didn't realise we flattened metadata as well. I'll need to update the docs if we're sticking with this.